data-loader/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "reaction-developer-tools",
```

Comment:
If we are going to make this a separate package like this, maybe we should call this `dataloader` or something and just make it a read separate package?